DifferentiationInterface/ext/DifferentiationInterfaceTapirExt/onearg.jl

```diff
@@ -5,25 +5,23 @@ end
 
 function DI.prepare_pullback(f, ::AutoTapir, x, dy)
```

Comment:
We could use this opportunity to store a little more in the `extras`, typically the objects that we have to allocate on the fly in `value_and_pullback`: `dy_righttype` and `dx_righttype`.
Same remark in the two-argument version

Reply:
If it's okay with you, I'd rather punt this for now. I don't have the time to figure out the best way to do this right now. Happy to revisit next week though.

Reply:
sounds good!